server/core/src/main/java/org/infinispan/server/core/configuration/ProtocolServerConfigurationChildBuilder.java

```diff
@@ -77,7 +77,10 @@
 
    /**
     * Sets the number of worker threads
+    *
+    * @deprecated since 14.0 with no replacement. No longer needed with non blocking core
     */
+   @Deprecated
    S workerThreads(int workerThreads);
```

Comment:
IMO, you can remove the methods. I don't see them as public API... I'm not expecting users to create the configuration programmatic to start the server in their applications :+1: